.vale/styles/RedHat/TermsWarnings.yml

```diff
@@ -35,13 +35,15 @@ swap:
   downgrade: upgrade|fallback|fall back|rollback|roll back
   entitlement: repository|subscription
   execute: run|issue|start|enter
+  hamburger: 'Do not use "hamburger". Use "the More options icon" or "the options icon".'
```

Comment:
```suggestion
kebab menu|kebab|hamburger: more options icon|options icon
```

Use a single rule with just the replavce terms so that users can click to replace.
![image](https://github.com/redhat-documentation/vale-at-red-hat/assets/74046732/c0fda236-c05f-4609-98fe-47849097d103)